nixos/modules/misc/ids.nix

```diff
@@ -295,6 +295,7 @@
       aria2 = 277;
       clickhouse = 278;
       rslsync = 279;
+      piwik = 280;
```

Comment:
Sounds good to me. If it turns out that a static uid makes more sense, it can always be added later (the other way 'round is a little harder).